commit a77adbe8d56dcaf4a138aa4026134e2869aa61cb
Author: Andreas Boll <aboll@debian.org>
Date:   Mon Feb 11 14:00:49 2019 +0100

    Revert "Map evdev keycode KEY_BRIGHTNESS_CYCLE to XF86MonBrightnessCycle"
    
    This reverts commit ed67b29ecc639f1bafa9530cb9307fee57241458.
    
    XF86MonBrightnessCycle currently depends on an unreleased xorgproto.
    
    See also
    https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/issues/123
    https://gitlab.freedesktop.org/xorg/proto/xorgproto/commit/4e9f4e6136a7c35b0b3d341424eb4e0585f2f0a7
    
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921867
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921940
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922020

diff --git a/symbols/inet b/symbols/inet
index 8e65c5e5..4d774fac 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -212,7 +212,7 @@ xkb_symbols "evdev" {
     key <I246>   {      [ XF86WLAN              ]       };
 //  key <I249>   {      [ ]       }; // KEY_VIDEO_NEXT  --  drive next video source
 //  key <I250>   {      [ ]       }; // KEY_VIDEO_PREV  --  drive previous video source
-    key <I251>   {      [ XF86MonBrightnessCycle ]      }; // KEY_BRIGHTNESS_CYCLE  --  bright up, max++ == min
+//  key <I251>   {      [ ]       }; // KEY_BRIGHTNESS_CYCLE  --  bright up, max++ == min
 //  key <I252>   {      [ ]       }; // KEY_BRIGHTNESS_ZERO  --  brightness off
 //  key <I253>   {      [ ]       }; // KEY_DISPLAY_OFF  --  turn off display
     key <I254>   {      [ XF86WWAN              ]       }; // KEY_WWAN
commit 0ea1d601ee56e1eb81be5f0a740e1380735e11ac
Author: Andreas Boll <aboll@debian.org>
Date:   Mon Feb 11 14:48:08 2019 +0100

    Revert "Map evdev keycode KEY_KEYBOARD to XF86Keyboard"
    
    This reverts commit d0e5e63cf59be444bec0e5b11838a9abce45226d.
    
    This patch breaks keyboard layouts in XWayland applications when having several
    keyboard layouts enabled.
    
    See also https://bugzilla.redhat.com/show_bug.cgi?id=1587998
    
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921867
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921940
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922020
    
    Drop this revert once we have a newer xkbcomp (>> 1.4.2) and bump
    dependency on x11-xkb-utils.

diff --git a/keycodes/evdev b/keycodes/evdev
index 90b8278d..f2630ffe 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -285,7 +285,6 @@ default xkb_keycodes "evdev" {
 	<I255> = 255;	// #define KEY_RFKILL              247
 
 	<I372> = 372;   // #define KEY_FAVORITES           364
-	<I374> = 374;   // #define KEY_KEYBOARD            366
 
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
diff --git a/symbols/inet b/symbols/inet
index 4d774fac..469a7d65 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -219,7 +219,6 @@ xkb_symbols "evdev" {
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
     key <I372>  {       [ XF86Favorites          ]       };
-    key <I374>  {       [ XF86Keyboard           ]       };
 
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
commit 3dd87655fd92a375ada3fc6d8cfe90d431baecaa
Author: Andreas Boll <aboll@debian.org>
Date:   Mon Feb 11 14:55:34 2019 +0100

    Revert "Map evdev keycode KEY_FAVORITES to XF86Favorites"
    
    This reverts commit e6103610a2ddae633e2715c9f1da86e79b5256dd.
    
    This patch breaks keyboard layouts in XWayland applications when having
    several keyboard layouts enabled.
    
    See also https://bugzilla.redhat.com/show_bug.cgi?id=1587998
    
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921867
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921940
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922020
    
    Drop this revert once we have a newer xkbcomp (>> 1.4.2) and bump
    dependency on x11-xkb-utils.

diff --git a/keycodes/evdev b/keycodes/evdev
index f2630ffe..51a6e26d 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -284,8 +284,6 @@ default xkb_keycodes "evdev" {
 	<I254> = 254;	// #define KEY_WWAN                246
 	<I255> = 255;	// #define KEY_RFKILL              247
 
-	<I372> = 372;   // #define KEY_FAVORITES           364
-
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
 	<MDSW> =   203;
diff --git a/symbols/inet b/symbols/inet
index 469a7d65..ef0635a7 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -218,8 +218,6 @@ xkb_symbols "evdev" {
     key <I254>   {      [ XF86WWAN              ]       }; // KEY_WWAN
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
-    key <I372>  {       [ XF86Favorites          ]       };
-
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
     key <FK15>   {      [ XF86Launch6       ]       };
